src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -176,6 +176,29 @@ public StudyController(StudyService studyService, NetworkStoreService networkSto
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(networkUuid.flatMap(networkStoreService::getNetworkVoltageLevels));
     }
 
+    @GetMapping(value = "/studies/{studyUuid}/network/{variantNum}/voltage-levels/{voltageLevelId}/buses")
+    @Operation(summary = "get buses the for a given network and a given voltage level")
+    @ApiResponse(responseCode = "200", description = "The voltage level list of the network")
```

Comment:
typo : 
"The **voltage level** list of the network"
"The **buses** list of the network"

Reply:
Corrected